frontend/public/components/factory/table-filters.ts

```diff
@@ -277,6 +278,12 @@ export const tableFilters: TableFilterMap = {
   'image-name': (str, imageManifestVuln) => fuzzyCaseInsensitive(str, imageManifestVuln.spec.image),
   vulnerability: (str, imageVulnerability) =>
     fuzzyCaseInsensitive(str, imageVulnerability.vulnerability.name),
+
+  // Filter installed operators by text match - name or displayName
```

Comment:
The comment doesn't match the code. It looks like we don't search by name, only display name. (I'd probably err on the side of matching both.)